### PR TITLE
ANC Subsequent visits - Outside Care activity

### PR DESCRIPTION
**Pull request by anvmn**
#257

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-05-18 12-13

**Comment by anvmn** [comment]
Peek 2022-05-18 20-59

**Comment by anvmn** [comment]
@mariano-dagostino please review.
Note there were some requierments changes, which were implemented at #281.

File: client/src/elm/Backend/PrenatalEncounter/Types.elm [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    | DiagnosisMolarPregnancy
+    | DiagnosisPlacentaPrevia
+    | DiagnosisPlacentalAbruption
+    | DiagnosisUterineRupture
+    | DiagnosisObstructedLabor
+    | DiagnosisPostAbortionSepsis
+    | DiagnosisEctopicPregnancy
+    | DiagnosisPROM
+    | DiagnosisPPROM
+      --  Diagnosed by dangher signs and is an emergency referral diagnosis.
```
Comment (mariano-dagostino) [inline client/src/elm/Backend/PrenatalEncounter/Types.elm:36]: I know this code was moved around, but seems a typo in `danger` no?

**Review by mariano-dagostino — APPROVED** [review]
@anvmn I was not able to fully review all this code. At a high level checked I don't see any red flag so far, just a minor typo in a comment. Feel free to assign this for another deep to someone else  in case you consider it pertinent.

File: client/src/elm/Backend/PrenatalEncounter/Types.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Backend/PrenatalEncounter/Types.elm:36]: I removed this comment in a subsequent PR, so, no need to worry about it.

**Pull request merged**